CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.2.2)
 project(leatherman VERSION 0.12.0)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+# Populate locale install location
+configure_file(cmake/leatherman.cmake.in "${CMAKE_CURRENT_SOURCE_DIR}/cmake/leatherman.cmake" @ONLY)
```

Comment:
I don't see the others being used during the Leatherman build process, so leatherman.cmake might need to go in a subdirectory of PROJECT_BINARY_DIR so you can safely add it to CMAKE_MODULE_PATH.